### DISCUSSION TITLE
Link room inside a Spoke scene, "visit room" does not appear.

### DISCUSSION THREAD

**Discussion by fenrir-liutiecheng** [mozilla/hubs#3217] [Hubs Cloud Questions] [answered]
I Want to link multiple rooms together. To do so, copy the link to the Hubs room I want users to move to, then go into the Spoke project, and press CTRL+V. A link （https://fenrir-hub.com/u7D6HyV/carefree-eminent-territory/）to the other room is appear, but  "visit room" not  appear.

![visitroom](https://user-images.githubusercontent.com/57246526/97067460-fa8f6280-15ef-11eb-8191-458e17b33710.jpg)

**Comment by robinkwilson** [comment, marked as answer]
I tested this myself on my Hubs Cloud instance and on hubs.mozilla.com, as long as the link is **a room on the same Hubs Cloud server**, it *should* say visit room. However if I link a room from hubs.mozilla.com on my Hubs Cloud instance, it will show "open link"/"refresh" buttons. 

Do you have a custom client deployed? You may need to merge the latest hubs-cloud branch into your custom client and deploy again.

If this is still a problem, you may want to make an issue in the hubs repo: https://github.com/mozilla/hubs/issues

Let me know if you're still having issues!

**Reply by fenrir-liutiecheng** [reply]
deploy again and it was worked!
thanks again!
